packages/gallery/src/components/gallery/proGallery/slideshowView.js

```diff
@@ -913,6 +932,47 @@ class SlideshowView extends GalleryComponent {
     return {arrowRenderer, navArrowsContainerWidth, navArrowsContainerHeight}
   }
 
+  // Function that checks if the nav arrows parent-container is large enough for them
+  arrowsWillFitPosition = () => {
+    const {
+      isSlideshow,
+      slideshowInfoSize,
+      arrowsVerticalPosition,
+      textBoxHeight,
+    } = this.props.options;
+    const { height } = this.props.container;
+    // Calc of Nav arrows container's height
+    const { navArrowsContainerHeight } = this.getArrowsSizeData();
+    let parentHeight;
+    const infoHeight = isSlideshow ? slideshowInfoSize : textBoxHeight;
+    switch (arrowsVerticalPosition){
+      case GALLERY_CONSTS.arrowsVerticalPosition.INFO_CENTER:
```

Comment:
Please change this `switch` to a map. Ask @yuvalab about it (she just refactored something similar now)